src/utils/dateFilter.ts

```diff
@@ -0,0 +1,8 @@
+import Vue from "vue"
+import moment from 'moment'
+
+Vue.filter('formatDate', function(value:string) {
```

Comment:
For added benefit we could include a second optional parameter that allows a user to specify the format, otherwise default to a reasonable format?